app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<div>
+<div>
+<h4> Preparation time: <%= @recipe.preparation_time %> </h4>
+<h4> Cooking time: <%= @recipe.cooking_time %> </h4>
+<p>Steps: <%= @recipe.description %> </p>
+</div>
+<div>
+<button> Generate shopping list </button>
+<button> Add ingredient</button>
+</div>
+<div>
+<table>
+<tr>
```

Comment:
Kindly, ensure that you always indent your code as it makes it easy to read and also easy to debug in case of any arising error. So please look into it